discovery-infra/tests/base_test.py

```diff
@@ -59,14 +59,18 @@ def cluster(self, api_client, request, nodes):
 
         def get_cluster_func(cluster_name: Optional[str] = None,
                              additional_ntp_source: Optional[str] = consts.DEFAULT_ADDITIONAL_NTP_SOURCE,
-                             openshift_version: Optional[str] = env_variables['openshift_version']):
+                             openshift_version: Optional[str] = env_variables['openshift_version'],
+                             user_managed_networking=False,
+                             high_availability_mode="Full"):
```

Comment:
should `high_availability_mode="Full"` be the default for all non-sno deployements?

Reply:
Yes